go.mod

```diff
@@ -62,10 +62,12 @@ require (
 	github.com/uber/jaeger-client-go v2.29.1+incompatible
 	github.com/vbauerster/mpb/v6 v6.0.4
 	github.com/vishvananda/netlink v1.1.1-0.20210330154013-f5de75959ad5
+	github.com/xi2/xz v0.0.0-20171230120015-48954b6210f8
```

Comment:
(A drive-by comment, I am not reviewing this now:) We already depend on github.com/ulikunitz/xz , could that one be used instead?

Reply:
Thanks for the feedback Miloslav

I missed the transitive dep there. It can definitely be changed. The reason I had gone with xi2 was that it is the closest to native xz utils in performance since it is a port of XZ Embedded (although still 30-50% slower in my tests). The ulikunitz/xz impl did poorly. For example, in a  300mb file it was 3x slower than xz (with buffering the input, if you dont it's an order of magnitude slower). On the other hand, ulikunitz is more actively maintained.  Although the format is fairly old and not evolving.  Since currently the image is only 35 megs, changing to ulikunitz only adds a few seconds. ; however, if the image was later improved to save the dnf time, the impact might be more noticeable since it will be a lot bigger.  Happy to switch it now with a note to maybe revisit later.  



`
xz utils
-------
22.292901
22.299508
22.255975
22.344327
22.293239

Average: 22.297190

xi2
---
31.553733
31.636368
31.582733
31.553733
31.486858

Average: 31.562685

ullikunitz
---------
64.670542
64.710961
64.545972
64.647141
64.679956

Average: 64.650914
`

Reply:
> Happy to switch it now with a note to maybe revisit later.

That would great, thank you!  We're usually careful with adding new dependencies since it implies work for downstream packagers when packaging guidelines demand de-vendoring each and every dependency (e.g., Debian).

Reply:
sounds good will do